modules/exploits/example_webapp.rb

```diff
@@ -168,6 +166,22 @@ def exploit
       'version' => '1.0',
       'uri' => '/' + ('../' * 560) + "\xcc\xcc\x90\x90" + '.smi'
     }, datastore['HttpClientTimeout'])
+
+    # example of sending a MIME message
+    data = Rex::MIME::Message.new
+    # https://github.com/rapid7/rex-mime/blob/master/lib/rex/mime/message.rb
+    data.add_part("upload-attachment", nil, nil, "form-data; name=\"action\"")
```

Comment:
Would you mind setting the third argument here for the `transfer_encoding` to `binary`. We've run into problems in the past where the default value will cause binary data to be corrupted on transfer. Calling it "upload-attachment" kind of makes it seem like you're trying to show binary data? I'm not entirely sure but either way a text example and a binary example would be helpful.

```ruby
      data.add_part(file_contents, 'application/octet-stream', 'binary', "form-data; name=\"file\"; filename=\"uploaded.bin\"")
````

Reply:
entirely reasonable